thaw/src/avatar/mod.rs

```diff
@@ -144,3 +152,16 @@ impl AvatarShape {
         }
     }
 }
+
+#[test]
+fn test_initials_name() {
+    assert_eq!(initials_name("Jane Doe".into()), "JD".to_string());
+    assert_eq!(initials_name("Ben".into()), "B".to_string());
+    assert_eq!(
+        initials_name("ÇFoo Bar 1Name too ÉLong".into()),
+        "ÇÉ".to_string()
+    );
+    assert_eq!(initials_name("ﬄ ß".into()), "FFLSS".to_string());
```

Comment:
This could also be changed to return `FS` if wanted

Reply:
Yes. The string length should not exceed 2.

Reply:
Changed 👍